### PR TITLE
[2.13.x] DDF-3903 The IdP should add its entity ID in the NameQualifier field of all Persistent Identifiers

### PR DESCRIPTION
**Pull request by blen-desta**
#### What does this PR do?
Backport of https://github.com/codice/ddf/pull/3354
The IdP should add its entity ID in the NameQualifier field of all Persistent Identifiers

#### Who is reviewing it? 
@bantillo @brjeter @bakejeyner 

#### Ask 2 committers to review/merge the PR and tag them here.
@coyotesqrl
@shaundmorris

#### How should this be tested?
Run the [SAML-CTK](https://github.com/codice/saml-conformance) tests against it.

#### What are the relevant tickets?
[DDF-3903](https://codice.atlassian.net/browse/DDF-3903)

#### Checklist:
- [ ] Documentation Updated
- [x] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by blen-desta** [comment]
build now

**Comment by blen-desta** [comment]
Unrelated failures
build now

**Comment by blen-desta** [comment]
```
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.6.CODICE:yarn 
(yarn install) on project ui: Failed to run task: 'yarn install --mutex network' failed.
org.apache.commons.exec.ExecuteException: Process exited with an error: 1
```
build now

File: features/security/src/main/feature/feature.xml [hunk 1]
```diff
@@ -565,6 +565,8 @@
         <feature>security-handler-saml</feature>
         <feature>session-invalidator</feature>
         <bundle>mvn:ddf.security.idp/security-idp-plugin-audiencerestriction/${project.version}</bundle>
+        <bundle>mvn:ddf.security.idp/security-idp-plugin-nameidpolicy/${project.version}</bundle>
```
[the remark this answers is not in the document]
Comment (blen-desta) [reply features/security/src/main/feature/feature.xml]: Yes I just moved it up

**Comment by Kjames5269** [comment]
Hero'd Successfully 🎉

**Pull request merged**